include/Player.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "Prelude.h"
+#include "Shuffle.h"
+
+#define MAX_NUMBER_OF_BLINDFOLDS 6
```

Comment:
A better way to do this would be to make a struct with all of these things inside, and then have it be accessible through the `AccessUnique` family of accessors wherever you need them. Also, why is the NUMBER_OF_BLINDFOLDS greater than the MAX_NUMBER_OF_BLINDFOLDS?